pkg/agent/host_configuration_unix.go

```diff
@@ -0,0 +1,133 @@
+// Copyright 2019 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// +build linux darwin
```

Comment:
Sure, I will remove it. 

Besides, I have one question about expected OS types for files with a suffix "_unix", shall we also support darwin for these files or only linux? @tnqn

Reply:
I think it depends on whether the code can work on darwin or not. After switching my IDE OS to darwin, it can't work because netlink doesn't implement the functions on darwin. So I think this should be linux only.

Reply:
Done.